packages/blade/src/components/Button/BaseButton/getBaseButtonStyles.ts

```diff
@@ -6,12 +6,15 @@ const getBaseButtonStyles = ({
   activeColor,
   buttonHeight,
   buttonSpacing,
+  isFullWidth,
```

Comment:
so so beautiful. Just pass the prop down to one single function, and things work out of the box on both platforms. ❤️. Any addition/deletion is just central